tg_bot.py

```diff
@@ -34,9 +34,17 @@ def start(self, update, context):
                     pass
                 url = 'https://twitter.com/' + \
                     username + '/status/' + str(line)
-                keyboard = [[InlineKeyboardButton(
-                    "Retweet", callback_data=int(line))]]
-                reply_markup = InlineKeyboardMarkup(keyboard)
+                tg_id = int(update.message.from_user.id)
+                
+                if tg_id in config.tg_admin_id:
+                    keyboard = [[InlineKeyboardButton(
+                        "Retweet", callback_data=int(line))]]
+                    reply_markup = InlineKeyboardMarkup(keyboard)
```

Comment:
For admins we can have both retweet ans view button can be displayed

Reply:
Okay will add that